src/web/components/ReaderRevenueLinks.tsx

```diff
@@ -11,6 +11,8 @@ import { textSans, headline } from '@guardian/src-foundations/typography';
 import { from, until } from '@guardian/src-foundations/mq';
 
 import { shouldHideSupportMessaging } from '@root/src/web/lib/contributions';
+import {fetchTickerData, TickerCountType, TickerData} from "@root/src/lib/fetchTickerData";
```

Comment:
looks like prettier is not working on `pre-push`, worth running it and fixing the styling diffs